lumicks/pylake/kymo.py

```diff
@@ -117,6 +134,19 @@ def pixel_time_seconds(self):
         """Pixel dwell time in seconds"""
         return (self.timestamps[1, 0] - self.timestamps[0, 0]) / 1e9
 
+    def _line_timestamp_ranges(self, exclude=True):
```

Comment:
Any reason you'd prefer keeping this private? It seems like something that would benefit users as well?